tests/new_api/test_device.py

```diff
@@ -259,7 +259,11 @@ def test_preprocess(self, adjoint):
         expected_program.add_transform(validate_measurements, name=device.name)
         expected_program.add_transform(validate_observables, accepted_observables, name=device.name)
         expected_program.add_transform(validate_device_wires, device.wires, name=device.name)
-        expected_program.add_transform(mid_circuit_measurements, device=device)
+        expected_program.add_transform(
+            mid_circuit_measurements,
+            device=device,
+            mcm_config={"postselect_shots": None, "mcm_method": None},
```

Comment:
Sorry @mudit2812! There is still some discussion on the name for `postselect_shots`, but I hope to resolve that today.

Reply:
Thanks for letting me know!